src/styles/theme/defaultTheme.ts

```diff
@@ -369,7 +369,6 @@ export const defaultTheme = createTheme({
       styleOverrides: {
         root: ({ theme }) => ({
           borderRadius: 12,
-          boxShadow: 'none',
```

Comment:
TODO: i have to check how it affected other cards

Reply:
It will impact cards in search results.

Reply:
I've been asked to revert this :shrug:

Reply:
wait or was this the revert requested, referring to something that added it earlier?